### PR TITLE
Added Endpoint to delete User with its id

### PR DESCRIPTION
**Pull request by anavduo**
Added method delete for User, added a UserController, UserService and implementation

### CODE REVIEW

File: src/main/java/com/alkemy/ong/service/impl/IUserService.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class IUserService implements UserService {
+    @Autowired
+    private UserRepository userRepo;
+    @Override
+    public void deleteUserById(String id) {
+
```
Comment (Lucasfranco1) [inline src/main/java/com/alkemy/ong/service/impl/IUserService.java:15]: El espacio este Ani, me parece que está de más.

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/service/impl/IUserService.java:15]: comment the methods at the beginning with what it does

Comment (anavduo) [reply src/main/java/com/alkemy/ong/service/impl/IUserService.java]: Comment added and blank space deleted, thanks fellows

**Review by juan1977lopez — APPROVED** [review]
Se aprueba PR. Buenos comentarios de los compañeros.

**Pull request merged**